### ISSUE TITLE
Window focus() fails

### ISSUE THREAD

**Issue by RalphMickelson** [kasper/phoenix#36]
The following code looks for a window containing the given name, and tries to bring it to front. The focus() function appears to always return 0/false.
```
log = function(x) {
  return Phoenix.log(x);
};

var findWindow = function(name) {
  var i, len, winList, win; 
  winList = Window.visibleWindows();
  for (i = 0, len = winList.length; i < len; i++) {
    win = winList[i];
    log(win.title());
    if (win.title().indexOf(name) > -1) {
      log(win.title());
      log(win.focus());
    }
  }
};

**Comment by shayne** [comment]
Running into `focus()` issues myself. It appears as if `focus()` on a `Window` only works if the window you want to focus is of the same app. *Example: I can use key bindings to cycle focus between iTerm windows (tested on a single screen), but if I call `focus()` on a Finder `Window` the `focus()` call fails.

KP?

**Comment by shayne** [comment]
The following still results in `false` response from `focus()` on `App`:
```javascript
if (win.app().hash() !== nextWin.app().hash()) {
  return nextWin.app().focus();
}
```

**Comment by kasper** [comment]
Bugger, what a oversight. Fixed in ae20e1f.
